OrbitGl/GpuTrack.cpp

```diff
@@ -217,7 +217,7 @@ std::string GpuTrack::GetSwQueueTooltip(const TimerInfo& timer_info) const {
       "<br/>"
       "<b>Submitted from thread:</b> %s [%d]<br/>"
       "<b>Time:</b> %s",
-      Capture::GThreadNames[timer_info.thread_id()].c_str(),
+      Capture::capture_data_.GetThreadName(timer_info.thread_id()).c_str(),
```

Comment:
Previously like this but you don't need `c_str()` in `StrFormat` (also below).

Reply:
Done. Thanks